### DISCUSSION TITLE
Welcome to rrdtool-1.x Discussions!

### DISCUSSION THREAD

**Discussion by oetiker** [oetiker/rrdtool-1.x#1157] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheWitness** [comment]
Congrat's on the new release @oetiker!  Glad to see it.  Best tool ever for TSDB IMHO.
